db.py

```diff
@@ -10,16 +10,16 @@
 import csl_reader
 
 LOGGER = logging.getLogger('pkt.funder.db')
-SEED = ('client ancient calm uncover opinion coil priority misery empty favorite moment myth')
-XPUB = 'tpubD6NzVbkrYhZ4XMSG7EWChwJXwfByid9TdZRVaej1rpDTHV3WamyuApceF5DDZXetx8kbH82NouoazYqPeCEZWWeXHZ1do5LBCe5xMcZYeGe'
+DEBUG = bool(os.environ.get('PAKET_DEBUG'))
```

Comment:
In the context of this module, this only indicates that we are using testnet, right?

If so, I'm considering changing the name of the constant - something like:
```python
TESTNET = bool(os.environ.get('PAKET_DEBUG'))```

Not sure about it.

Reply:
No, in context of this module it is also used to indicate debug mode, to prevent using debug-only function - db.get_users()

Reply:
:+1: